README.md

```diff
@@ -71,26 +73,99 @@ Django application for the DDP platform's management backend. Exposes API endpoi
 
 -   `pip install -r requirements.txt`
 
+### Step 2: Create the .env file
+
 -   create `.env` from `.env.template`
 
+### Step 3: Create SQL Database
+
 -   create a SQL database and populate its credentials into `.env`
 
+-   You can use a postgresql docker image for local development
+
+``` 
+docker run --name postgres-db -e POSTGRES_PASSWORD=<password> -p 5432:5432 -d <db name>
+
+```
+
+- Add the environment variable to .env
+
+```
+DBNAME=<db name>
+DBHOST=localhost
+DBPORT=5432
+DBUSER=postgres
+DBPASSWORD=<password>
+DBADMINUSER=postgres
+DBADMINPASSWORD=<password>
+
+```
+
+### Step 4: Install Airbyte
+-   Open a new terminal
 -   [Start Airbyte](https://docs.airbyte.com/deploying-airbyte/local-deployment) and populate connection info in `.env`
 
--   [Start Prefect](https://docs.prefect.io/latest/getting-started/installation/) and populate connection info in `.env`
+```
+AIRBYTE_SERVER_HOST=
+AIRBYTE_SERVER_PORT=
+AIRBYTE_SERVER_APIVER=
+AIRBYTE_API_TOKEN= <token> # base64 encryption of username:password. Default username and password is airbyte:password and token will be YWlyYnl0ZTpwYXNzd29yZA==
+AIRBYTE_DESTINATION_TYPES=
+```
+
+### Step 5: Install Prefect
+-   Open a new terminal. 
+
+-   Clone [Prefect Proxy](https://github.com/DalgoT4D/prefect-proxy)
```

Comment:
Since I updated the instructions in the prefect-proxy to include the cloning step, we could remove the instruction from here to avoid confusion.